src/commands/createStaticWebApp/createStaticWebApp.ts

```diff
@@ -22,6 +22,16 @@ export async function createStaticWebApp(context: IActionContext & Partial<ICrea
         node = await ext.tree.showTreeItemPicker<SubscriptionTreeItem>(SubscriptionTreeItem.contextValue, context);
     }
 
+    const folder: WorkspaceFolder = await getWorkspaceFolder(context);
+    const gitWorkspaceState: GitWorkspaceState = await getGitWorkspaceState(context, folder.uri);
+    // verifyRepoForCreation will set gitWorkspaceState.repo, but typing will yell if we don't set it here
+    gitWorkspaceState.repo = await verifyRepoForCreation(context, gitWorkspaceState, folder.uri);
+
+    await promptForDefaultBranch(context, gitWorkspaceState.repo);
+    context.telemetry.properties.cancelStep = undefined;
```

Comment:
I think it would be cleaner to put this line at the end of `promptForDefaultBranch` (I believe that would have the same effect?)

Reply:
Yeah, though I put it within createStaticWebApp because the cancelStep is related to the `staticWebApps.createStaticWebApp` specifically.